src/channels/pages/ChannelDetailsPage/ChannelDetailsPage.tsx

```diff
@@ -211,6 +218,13 @@ const ChannelDetailsPage = function <TErrors extends ChannelErrorFragment[]>({
                 : TransactionFlowStrategyEnum.CHARGE,
           });
         };
+
+        const handleAutomaticallyCompleteCheckoutsChange = () => {
+          set({
+            automaticallyCompleteCheckouts: !data.automaticallyCompleteCheckouts,
```

Comment:
question: why do you use `!`?

Reply:
Becasue its a checkbox, so it inverts data (false → true, true → false)
The handler doesnt receive the change, it uses current data to flip the state